### PR TITLE
Use an SSR friendly ID

### PR DESCRIPTION
**Pull request by msutkowski**
Addresses #25 and allows for usage in a SSR environment like next.js.

### CODE REVIEW

**Comment by msutkowski** [comment]
This is just an idea and probably doesn't need to be exported, but this is going to be more performant that a universal implementation of `performance.now`. @e271828- let me know if you have any thoughts, thanks.

**Comment by e271828-** [comment]
Adding @brdlyptrs for his thoughts. Seems reasonable to me.

**Comment by brdlyptrs** [comment]
This looks like a good solution. Let's import the package as a dependency, instead of copying the code @msutkowski.

**Comment by msutkowski** [comment]
@brdlyptrs Just a habit from a few projects I contribute to regarding further optimized bundle outputs and re-exporting these types of functions. In our internal projects, we try to inline simple libs like this to keep external dependencies down and to avoid potential build pipeline problems, but I think with this one that isn't really a concern :). I'll update the PR in a few.

**Comment by msutkowski** [comment]
@brdlyptrs So quick update, it looks like the bundle when requiring the `nanoid` lib isn't going to shake out the other exports it has so we'd be unnecessarily inflating the overall package size, so I'd recommend just keeping this function inlined. 

Another option would be setting up `rollup` for this repo. I could see if I could get the production build to shake out the rest of `nanoid`s imports and just leave it as a dev dependency.

Another option would be is that I could just rebuild this lib on top of [tsdx](https://github.com/jaredpalmer/tsdx) and introduce some steps for easy publishing and releases if you'd like. I know @e271828- mentioned you were probably holding off on TS for the time being, but I think with this kind of simple component the TS conversion is extremely minimal and doesn't need to support generic use cases. You'd also take the overhead out of builds and releases, but again, 100% up to you all. 

I'm happy to help with any of these options, just let me know what you want and what works best for your team :)

**Comment by brdlyptrs** [comment]
We also try to keep our dependencies to a minimum. `RollupJS` is used in some of are other repositories, so it may be worth while to consider it for this one. That being said, let's do the smallest change first. Thanks for helping out with this! @msutkowski

File: src/index.js [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
 
-// Create random ID
-const randomID = () => `${(Number(String(Math.random()).slice(2)) + Date.now() + Math.round(performance.now())).toString(36)}`;
+// Borrowed from https://github.com/ai/nanoid/blob/3.0.2/non-secure/index.js
+// This alphabet uses `A-Za-z0-9_-` symbols. A genetic algorithm helped
+// optimize the gzip compression for this alphabet.
+let urlAlphabet =
+  'ModuleSymbhasOwnPr-0123456789ABCDEFGHNRVfgctiUvz_KqYTJkLxpZXIjQW'
+
+export let nanoid = (size = 21) => {
```
Comment (brdlyptrs) [inline src/index.js:9]: Since this function and the above variable do not change, could you make them`const` instead of `let`.

Comment (msutkowski) [reply src/index.js]: All set.

**Comment by msutkowski** [comment]
@brdlyptrs No problem :)

**Pull request merged**